packages/ppx_regexp/ppx_regexp.0.5.0/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "Petter A. Urkedal <paurkedal@gmail.com>"
+authors: [
+  "Petter A. Urkedal <paurkedal@gmail.com>"
+  "Gabriel Radanne <drupyog@zoho.com>"
+]
+license: "LGPL-3.0-or-later WITH LGPL-3.0-linking-exception"
+homepage: "https://github.com/paurkedal/ppx_regexp"
+bug-reports: "https://github.com/paurkedal/ppx_regexp/issues"
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "dune" {>= "1.11"}
+  "ppxlib" {>= "0.9.0"}
+  "re" {>= "1.7.1"}
```

Comment:
```suggestion
  "re" {>= "1.7.2"}
```
Solves `Unbound module Re.Perl`. This is the version that introduced the Re. modules in place of Re_

Reply:
Indeed, thanks!